src/pt/mangalivre/build.gradle

```
@@ -2,8 +2,8 @@ ext {
     extName = 'Manga Livre'
     extClass = '.MangaLivre'
     themePkg = 'madara'
-    baseUrl = 'https://mangalivre.tv'
-    overrideVersionCode = 1
+    baseUrl = 'https://mangalivre.to'
+    overrideVersionCode = 3
```

Extension its still using the `.tv` domain in source class
```suggestion
    baseUrl = 'https://mangalivre.tv'
    overrideVersionCode = 2
```